R/getSettings.R

```diff
@@ -0,0 +1,66 @@
+# try to find the VSC settings file.
+# can be tricky for specific platforms, such as WSL,
+# where the settings are located on the window's side
+fileCon <- if (file.exists("~/.config/Code/User/settings.json")) {
```

Comment:
The problem mostly resides here as I'm not sure if there is a clean way to determine the path of the vscode config file under multiple scenarios: local instance, remote development, system installation, user installation, portable, etc.

Reply:
I think the best way is not let the R session to find the vscode settings file but vscode-R writes a JSON file in `~/.vscode-R/settings.json` where only `vsc.*` related settings are included. On configuration update of these settings, the file should be updated too.

Then `.vsc.attach()` should read `~/.vscode-R/settings.json` to apply the latest settings.

Reply:
Thanks, that's a great idea :)